Argo/Types/StandardTypes.swift

```diff
@@ -69,6 +69,20 @@ public func decodeObject<A where A: Decodable, A == A.DecodedType>(value: JSON)
   }
 }
 
+func decodedJSONForKey(json: JSON, key: String) -> Decoded<JSON> {
```

Comment:
we can't because it's being used in the `Operators.swift` file ... but it won't be visible to the public interface because it's internal by default